### ISSUE TITLE
Typo on speaker badge

### ISSUE THREAD

**Issue by DavidHernandez** [AsociacionDrupalES/DrupalCampSpain#249]
The speaker badge says `I'm speaker` but I think it should say `I'm a speaker`: https://2017.drupalcamp.es/pages/badges

**Comment by juampynr** [comment]
Thanks @DavidHernandez!

Actually, I think it's "I'm speaking", but we can confirm this with other event's badges.

**Comment by juampynr** [comment]
Yep, based on https://austin2014.drupal.org/spread-the-word.html, it's "I'm a speaker". cc @wildwildweb

**Comment by wildwildweb** [comment]
@juampynr @DavidHernandez Find attached corrected badges! 
![im-speaker-300x300](https://cloud.githubusercontent.com/assets/592608/25224550/501eef74-25f1-11e7-8df5-5668986debb5.jpg)
![im-speaker-1080x1080](https://cloud.githubusercontent.com/assets/592608/25224551/504704dc-25f1-11e7-92e9-174e25675ea1.jpg)
